pkg/services/ngalert/notifier/redis_peer.go

```diff
@@ -27,6 +27,7 @@ type redisConfig struct {
 	db       int
 	name     string
 	prefix   string
+	maxConns int
```

Comment:
If this is left unset, it won't crash, it'll just use the redis library's default. Not a dangerous change, and we don't need any special logic to detect zero below.